kubetest2-gke/deployer/up.go

```diff
@@ -164,6 +163,11 @@ func (d *Deployer) CreateCluster(project string, cluster cluster, subNetworkArgs
 		}
 	} else {
 		args = append(args, "--cluster-version="+d.Version)
+		releaseChannel, err := resolveReleaseChannelForClusterVersion(d.Version, locationArg)
+		if err != nil {
+			return err
```

Comment:
/hold

This should be a warning, since we also may want to test bleeding edge versions which aren't in any channel yet.

Reply:
Done.